bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -1223,6 +1232,52 @@ Error LinuxKernelRewriter::readBugTable() {
   return Error::success();
 }
 
+/// find_bug() uses linear search to match an address to an entry in the bug
+/// table. Hence, there is no need to sort entries when rewriting the table.
+/// When we need to erase an entry, we set its instruction address to zero.
+Error LinuxKernelRewriter::rewriteBugTable() {
+  if (!BugTableSection)
```

Comment:
is this possible at all? Can you have a binary without bug table?

Reply:
Yes. If `CONFIG_GENERIC_BUG` is not set in the kernel config, the section will not be emitted.